Program.cs

```diff
@@ -823,8 +848,12 @@ public static void attack()
         public static void NameWelcome()
         {
             Console.WriteLine("Welcome To The Game");
-            Console.Write("Please Enter Your Player Name: ");
+            Console.Write("Please Enter Your Player Name or type load to load you last save: ");
             name = Console.ReadLine();
+            if(name=="load")
```

Comment:
I think "if(name=="load")" is not recognized. When I enter "load", it shows that the player name I entered before is changed to "load